lib/config/nav_bar_config.dart

```diff
@@ -43,4 +44,16 @@ extension IsRouteATabViewX on PageRouteInfo<dynamic> {
   NavBarEnum? get tabBarEnum => routeName.tabBarEnum;
 
   bool get isTabView => routeName.isTabView;
+
+  String? niceRouteName(BuildContext context) {
+    return switch (routeName) {
+      HomeRoute.name => context.localize.home_screen,
+      NavigationTabRoute.name => context.localize.other_view,
+      DepartmentsRoute.name => context.localize.departments,
+      SksMenuRoute.name => context.localize.sks_menu,
+      ScienceClubsRoute.name => context.localize.scientific_cirlces,
+      GuideRoute.name => context.localize.guide,
+      _ => null,
+    };
+  }
```

Comment:
There are more routes to be added in here. 

![image](https://github.com/user-attachments/assets/975eec95-d4cb-4a62-9c67-999711cfbe81)

(problem appears) we go to science clubs from detailed department info.

Reply:
Good catch! Hella good catch. This actually prolly will be solved by adding missing routes to nice route name